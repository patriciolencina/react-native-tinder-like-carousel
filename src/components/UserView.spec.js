import React from 'react';
import {shallow} from 'enzyme';
import UserView from './UserView';

describe('UserView', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      expect(shallow(<UserView />)).toMatchSnapshot();
    });
  });
});
