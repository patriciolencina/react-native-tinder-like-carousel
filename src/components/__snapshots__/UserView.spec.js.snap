// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserView Rendering should match to snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "backgroundColor": "white",
      "borderRadius": 8,
      "marginHorizontal": 10,
      "paddingTop": 30,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#F9F9F9",
        "borderBottomColor": "#C5C5C5",
        "borderBottomWidth": 1,
        "height": 100,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "borderColor": "#C5C5C5",
        "borderRadius": 100,
        "borderWidth": 1,
        "height": 200,
        "justifyContent": "center",
        "width": 200,
      }
    }
  >
    <Image
      source={
        Object {
          "uri": undefined,
        }
      }
      style={
        Object {
          "backgroundColor": "gray",
          "borderRadius": 90,
          "height": 180,
          "width": 180,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "column",
        "justifyContent": "center",
        "marginHorizontal": 40,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#999999",
          "fontSize": 17,
          "marginTop": 30,
        }
      }
    >
      My address is
    </Text>
    <Text
      style={
        Object {
          "color": "black",
          "fontSize": 22,
          "marginBottom": 20,
          "textAlign": "center",
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "stretch",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "marginBottom": 30,
        "marginHorizontal": 60,
      }
    }
  >
    <Icon
      allowFontScaling={false}
      color="black"
      name="md-person"
      size={34}
    />
    <Icon
      allowFontScaling={false}
      color="black"
      name="ios-calendar"
      size={34}
    />
    <Icon
      allowFontScaling={false}
      color="black"
      name="ios-map"
      size={34}
    />
    <Icon
      allowFontScaling={false}
      color="black"
      name="md-call"
      size={34}
    />
    <Icon
      allowFontScaling={false}
      color="black"
      name="ios-lock"
      size={34}
    />
  </View>
</View>
`;
